services/credential-server-ui/src/pages/RevocationPage.tsx

```diff
@@ -48,11 +51,18 @@ const RevocationPage: React.FC = () => {
   };
 
   const handleRevokeCredential = async (values: any) => {
-    await axios.post(`${config.endpoint}${config.path.revokeCredential}`, {
-      credentialId: values.selectedCredential,
-      holder: values.selectedContact
-    });
-    await handleGetContactCredentials(values.selectedContact);
+    try {
+      const response = await axios.post(`${config.endpoint}${config.path.revokeCredential}`, {
+        credentialId: values.selectedCredential,
+        holder: values.selectedContact
+      });
+      if (response.status === 200) {
+        setIsRevokeCredentialSuccess(true);
```

Comment:
quick one, can we set it back to `false` when we try to revoke - so we know that we're actively trying to revoke a second credential? (and in the other files for issuance etc)